SplitIt/app/src/main/java/project/cs495/splitit/MainActivity.java

```diff
@@ -58,6 +58,9 @@
     private FirebaseAuth auth;
     private int fabState = 0;
     private String selectedGroupIdForReceipt;
+    private ImageButton fab_plus;
+    private ImageButton fab_scan_receipt;
+    private ImageButton fab_add_group;
```

Comment:
We moved away from doing ImageButtons. This looks like the result of merge conflict resolution. I do not think we need the ImageButtons at all.